lib/ansible/modules/cloud/amazon/cloudwatchlogs_log_group_metric_filter.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: cloudwatchlogs_log_group_metric_filter
+version_added: "2.10"
+author:
+  - "Markus Bergholz (@markuman)"
+short_description: Manage CloudWatch log group metric filter
+description:
+  - Create, modify and delete Cloudwatch log group metric filter.
+  - Cloudwatch log group metric filter can be use with I(ec2_metric_alarm).
```

Comment:
```suggestion
  - Cloudwatch log group metric filter can be use with M(ec2_metric_alarm).
```

See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation